content/en/docs/tutorials/stateless-application/guestbook.md

```diff
@@ -200,7 +200,7 @@ If you deployed this application to Minikube or a local cluster, you need to fin
 
 1. Run the following command to get the IP address for the frontend Service.
 
-       minikube service frontend --url
+       minikube service frontend --url (be sure to use a double hypen before "url" in case your browser is showing a single, copy and paste will fail)
```

Comment:
the command is not properly shown maybe because it is missing some markups ... how about try wrapping it into a block like this?

```
minikube service ...
```

Reply:
That makes far more sense than writing out a full sentence in () like I did, though. :p Mainly guess I'm not what would work best to display properly/what they'd like on the site and just wanted to bring attention to it.

Reply:
Thanks for letting us know there are defects, :). Please review #8694 and see if it fixes the output.